packages/blue-sdk/src/addresses.ts

```diff
@@ -427,6 +427,45 @@ const _unwrappedTokensMapping: Record<number, Record<Address, Address>> = {
     [_addressesRegistry[ChainId.BaseMainnet].testUsdc]:
       _addressesRegistry[ChainId.BaseMainnet].usdc,
   },
+  [ChainId.PolygonMainnet]: {
+    [_addressesRegistry[ChainId.PolygonMainnet].wNative]: NATIVE_ADDRESS,
+  },
+  [ChainId.ArbitrumMainnet]: {
+    [_addressesRegistry[ChainId.ArbitrumMainnet].wNative]: NATIVE_ADDRESS,
```

Comment:
wNative on arbitrum [is upgradable](https://arbiscan.io/address/0x82aF49447D8a07e3bd95BD0d56f35241523fBab1#code)